open_city_profile/settings.py

```diff
@@ -282,20 +283,24 @@
 # Month from which on the membership will last until the next year, instead of ending in the current year
 YOUTH_MEMBERSHIP_FULL_SEASON_START_MONTH = 5
 
+if not DEBUG:
+    audit_handler = {
+        "level": "INFO",
+        "class": "logging.FileHandler",
+        "filename": "./audit.log",
+    }
+else:
+    audit_handler = {
+        "level": "INFO",
+        "class": "logging.StreamHandler",
+        "stream": stdout,
+    }
 
-AUDIT_LOGGING_ENABLED = env.bool("AUDIT_LOGGING_ENABLED")
+AUDIT_LOGGING_ENABLED = True  # env.bool("AUDIT_LOGGING_ENABLED")
```

Comment:
Do we want to enable audit logging by default? If so maybe we could instead change the default value on line 46?

Reply:
No, we do not. My bad.